src/containers/edit-tutor-profile/NotificationContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import Box from '@mui/system/Box'
+import Typography from '@mui/material/Typography'
+import Switch from '@mui/material/Switch'
+
+import { useTranslation } from 'react-i18next'
+
+import SettingItem from '~/components/setting-item/SettingItem'
+import { styles } from '~/containers/edit-tutor-profile/NotificationContainer.style'
```

Comment:
organize imports and add space between them and component

Reply:
done